internal/store/config.go

```diff
@@ -19,11 +19,14 @@ import (
 	"fmt"
 	"time"
 
-	// this project.
+	// first-party project.
```

Comment:
```suggestion
	// third-party project.
```

Reply:
this is ok, because here are other packages in vanus